src/zenml/zen_server/deploy/helm/templates/server-db-pvc.yaml

```diff
@@ -0,0 +1,18 @@
+# if a storage class is set, create the pvc that will hold a backup of the database
+{{- if .Values.zenml.database.storageClass }}
+apiVersion: v1
+kind: PersistentVolumeClaim
```

Comment:
I think you'll find that there are problems if you mix helm-hook resources with regular resources. I'm not sure about volumes, but I think the same will happen here: the volume won't be created until _after_ the DB job runs successfully (which can't happen).

Reply:
Makes sense, good catch!